account_move_line_purchase_info/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Account Move Line Purchase Info",
     "summary": "Introduces the purchase order line to the journal items",
-    "version": "17.0.1.0.1",
+    "version": "17.0.2.0.0",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/account-financial-tools",
     "category": "Generic",
```

Comment:
BTW, why this module doesn't depend on `account_move_line_stock_info`?

Reply:
@MiquelRForgeFlow I don't see that it uses anything specific from account_move_line_stock_info